Blog SBOLinside announcement (#265)
SynBioDex/SbolStandardWebsite: content/post/SBOLinside-Announcement/index.md

---

> title: "SBOLinside goes online"

Reviewer: ```suggestion
title: "Introducing SBOLinside: Our New Video Podcast"
```

date: 2026-07-21
summary: >

>   We proudly announce SBOLinside, our first Videopodcast format in which we present insights to SBOL and SBOL Visual.

Reviewer: ```suggestion
  We proudly announce SBOLinside, our first video podcast, where we present insights into SBOL and SBOL Visual.
```

tags:
  - SBOL
  - SBOL Visual
  - Videopodcast
  - Community
---

> ![[SBOLinside_Banner.png]]

Reviewer: ```suggestion
{{< figure src="SBOLinside_Banner.png" title="" lightbox="true" >}}
```

> Next to the day-to-day editorial work for SBOL, we have been working on something special for you as the SBOL community. 

Reviewer: ```suggestion
Alongside our day-to-day editorial work, we have been developing something special for the SBOL community.
```

> We asked, how can we provide insights to SBOL and its organization and additionally allow the community to follow our current work. 

Reviewer: ```suggestion
We asked ourselves: how can we offer deeper insights into SBOL and our organisation while making it easier for the community to follow our ongoing work? 
```

Our answer is **SBOLinside**.

SBOLinside is a video podcast format specifically focusing on providing inside stories from our work on SBOL and our organization. 

> The format will include interviews with people from SBOL like the Steering Committee, the SBOL Chair, the Editors and many more. 

Reviewer: ```suggestion
The series will feature interviews with key figures driving the standard forward, including members of the Steering Committee, the Editors, the Chair, and many more.
```

> Besides, we will present what we are currently working on and how the community can engage to support the Synthetic Biology Open Language (SBOL). 

Reviewer: ```suggestion
We will also present what we are currently working on and share ways on how you can actively engage with and support the Synthetic Biology Open Language (SBOL). 
```

The episodes of SBOLinside will be released on our [YouTube channel](https://www.youtube.com/@syntheticbiologyopenlangua6204) and announced on [LinkedIn](https://www.linkedin.com/company/sbol/) and our [blog](https://sbolstandard.org/blog/).

> You want to learn about a particular aspect of SBOL? Reach out to us and we will be happy to consider your topic for an episode.

Reviewer: ```suggestion
Is there a specific aspect of SBOL you would love to learn more about? Reach out to us and we will be happy to consider your topic for a future episode.
```
